src/common/Core/ContextBridge.ts

```diff
@@ -28,6 +28,7 @@ export type ContextBridge = {
     getAvailableExtensions: () => ExtensionInfo[];
     getAvailableTerminals: () => Terminal[];
     getEnabledExtensions: () => ExtensionInfo[];
+    getEnvironmentVariable: (environmentVariable: string) => string;
```

Comment:
This returns potentially `undefined`:

```suggestion
    getEnvironmentVariable: (environmentVariable: string) => string | undefined;
```